src/main/java/banking/dao/AccountDAO.java

```diff
@@ -0,0 +1,155 @@
+package banking.dao;
+
+import banking.configuration.DBManager;
+import banking.model.Account;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class AccountDAO {
+
+    private final static String INSERT_INTO = "INSERT INTO account (number, pin) VALUES (?, ?)";
```

Comment:
The order is important, please use `private static final`.

Reply:
fixed

Reply:
I can still see `private final static`. Can you please fix that?

Reply:
fixed